course5/homework2/newton/newton.cc

```diff
@@ -0,0 +1,59 @@
+#include <iostream>
+#include <string>
+#include <time.h>
+
+// Newton method of sqrt. The stack frame was 48 for Debug mode and 64 for Release mode.
+
+int i = 0;
```

Comment:
Didn't I explicitly asked in the alignements not to use global variables?
Maybe I forgot...
Anyway, try again, without global variables!